Source/Fuse.Drawing.Surface/CoreGraphics/NativeSurface.uno

```diff
@@ -0,0 +1,48 @@
+using Uno;
+using Uno.Collections;
+using Uno.Compiler.ExportTargetInterop;
+using Fuse.Drawing.Primitives;
+
+namespace Fuse.Drawing
+{
+	extern(iOS||OSX)
+	class NativeSurface : CoreGraphicsSurface
+	{
+
+		IntPtr _cgContext = IntPtr.Zero;
+
+		public void Begin(IntPtr cgContext, float pixelsPerPoint)
+		{
+			// The CGContext is already scaled
+			_pixelsPerPoint = 1.0f;//pixelsPerPoint;
```

Comment:
Does this assume the pixelsPerPoint is the same as pixelsPerOSPoint?  I guess the entire native layer does that?